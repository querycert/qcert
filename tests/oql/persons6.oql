/* View definition, including dropping a view */
define t as Clients;

define s as select struct(name:c->name, purchased: p->name)
from c in t,
     p in Purchases
where p->cid = c->id
;

undefine t;

s
