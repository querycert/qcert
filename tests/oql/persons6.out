[{"name": "Tomatoe Paradise", "purchased": "Tomatoe"}, {"name": "Tomatoe Paradise", "purchased": "Potatoe"}, {"name": "Stillettos are Us", "purchased": "Stiletto"}]